api/app/signals/apps/api/v1/urls.py

```diff
@@ -26,6 +27,7 @@
 # Public API
 public_router = SignalsRouterVersion1()
 public_router.register(r'public/terms/categories', ParentCategoryViewSet, basename='category')
+# public_router.register(r'public/questions/categories', PublicQuestionViewSet, basename='question')
```

Comment:
If not used please remove instead of marking it as a comment

Reply:
check